momepy/functional/_elements.py

```diff
@@ -190,6 +195,17 @@ def enclosed_tessellation(
     single = unique[counts == 1]
     altered = unique[counts > 0]
 
+    if simplify:
+        if not SHPLY_GE_205:
+            raise ImportError(
+                "Coverage simplification requires shapely 2.0.5 or higher."
+            )
+        from shapely import coverage_simplify
+
+        ##simplify singles here and let the threads simplify the actual tesselations
+        simplified = coverage_simplify(enclosures.iloc[single], 1e-1)
```

Comment:
nope, we should not touch enclosures. They are not problematic and this will result in mismatch with streets.